mix.exs

```diff
@@ -35,7 +35,7 @@ defmodule Exq.Mixfile do
   defp deps do
     [
       { :uuid, ">= 1.1.0" },
-      { :redix, ">= 0.5.0"},
+      { :redix, ">= 0.6.1" },
```

Comment:
Is this our version of remix? if so I think we should add something like `0.6.1-ld` so that it won't conflict with the office `0.6.1`

Reply:
They reference the commit sha so it will match the correct version. Are you saying it's a conflict in reading it?